src/Atmos/Model/boundaryconditions.jl

```diff
@@ -77,27 +77,27 @@ struct NoFluxBC <: BoundaryCondition
 end
 
 function atmos_boundary_state!(nf::Union{NumericalFluxNonDiffusive, NumericalFluxGradient},
-                               bc::NoFluxBC, m::AtmosModel, stateP::Vars,
-                               auxP::Vars, nM, stateM::Vars, auxM::Vars, bctype,
+                               bc::NoFluxBC, m::AtmosModel, state⁺::Vars,
+                               aux⁺::Vars, n⁻, state⁻::Vars, aux⁻::Vars, bctype,
                                t, _...)
-  FT = eltype(stateM)
-  stateP.ρ = stateM.ρ
+  FT = eltype(state⁻)
+  state⁺.ρ = state⁻.ρ
   if typeof(nf) <: NumericalFluxNonDiffusive
-    stateP.ρu -= 2 * dot(stateM.ρu, nM) * SVector(nM)
+    state⁺.ρu -= 2 * dot(state⁻.ρu, n⁻) * SVector(n⁻)
   else
-    stateP.ρu -=  dot(stateM.ρu, nM) * SVector(nM)
+    state⁺.ρu -=  dot(state⁻.ρu, n⁻) * SVector(n⁻)
   end
 end
 
 function atmos_boundary_state!(::NumericalFluxDiffusive, bc::NoFluxBC,
-                               m::AtmosModel, stateP::Vars, diffP::Vars,
-                               auxP::Vars, nM, stateM::Vars, diffM::Vars,
-                               auxM::Vars, bctype, t, _...)
-  FT = eltype(stateM)
-  stateP.ρ = stateM.ρ
-  stateP.ρu -= dot(stateM.ρu, nM) * SVector(nM)
+                               m::AtmosModel, state⁺::Vars, diff⁺::Vars,
+                               aux⁺::Vars, n⁻, state⁻::Vars, diff⁻::Vars,
+                               aux⁻::Vars, bctype, t, _...)
+  FT = eltype(state⁻)
+  state⁺.ρ = state⁻.ρ
+  state⁺.ρu -= dot(state⁻.ρu, n⁻) * SVector(n⁻)
   
-  fill!(getfield(diffP, :array), FT(0))
+  fill!(getfield(diff⁺, :array), FT(0))
```

Comment:
@jkozdon @fxgiraldo out of curiousity: should we be setting `diff⁺ = 0` or `diff⁺ = -diff⁻`

Reply:
I believe that we want a zero Neumann BC here, so zero would be correct. If you were setting a Dirichlet you would set `diff⁺ = -diff⁻`

Reply:
> If you were setting a Dirichlet you would set `diff⁺ = -diff⁻`

Why not `diff⁺ = diff⁻`?